Tests/LibTimeZone/TestTimeZone.cpp

```diff
@@ -183,7 +183,7 @@ TEST_CASE(get_named_time_zone_offsets)
 
 TEST_CASE(time_zone_from_string)
 {
-    EXPECT_EQ(TimeZone::time_zone_from_string("UTC"sv), TimeZone::TimeZone::UTC);
```

Comment:
Not actually sure how this ever compiled. It was added in 205d63c3f05, but the enum when ENABLE_TIME_ZONE_DATA is OFF is in Userland/Libraries/LibTimeZone/TimeZone.cpp, so it was never visible here?